deny.toml

```diff
@@ -0,0 +1,214 @@
+# This section is considered when running `cargo deny check advisories`
+# More documentation for the advisories section can be found here:
+# https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html
+[advisories]
+# The path where the advisory database is cloned/fetched into
+db-path = "/cache/cargo/advisory-db"
```

Comment:
I added this: https://github.com/radicle-dev/radicle-link/pull/75/files#diff-8d3288bb6c44ba57934db1e4fa9c1d97R10-R13

So one can run `cargo deny` locally without any surprises